### ISSUE TITLE
Firefox support

### ISSUE THREAD

**Issue by MaxDesiatov** [swiftwasm/swiftwasm-pad#1]
Looks like this doesn't fully work on Firefox, I get these errors:

```
library.so.wasm was downloaded main.js:1:388339

main.js:1:388339
Info: Reading /tmp/main.o main.worker.js:1:187727

main.worker.js:1:187727
Warning: Custom section 'target_features' is currently not supported main.worker.js:1:187727

main.worker.js:1:187727
Info: Reading /tmp/library.so.wasm main.worker.js:1:187727

main.worker.js:1:187727
Warning: Linking subsection 'comdatInfo' is not supported now main.worker.js:1:187727

main.worker.js:1:187727
Warning: Custom section 'producers' is currently not supported main.worker.js:1:187727

main.worker.js:1:187727
TypeError: cannot transfer WebAssembly/asm.js ArrayBuffer
```

**Comment by kateinoigakukun** [comment]
Thank you for your feedback. This is fixed by 4c14d5e. Please try again 🙏

**Comment by MaxDesiatov** [comment]
Awesome, thanks for the quick fix 👍
